status-im/status-react
# Show token requirements only for token gated channels

fixes #19350

### Summary
Differentiating the token-gated communities and channels when showing the token requirements bottom-sheet. -
Added:
- hiding token requirements from **community** context menu when community has no "role-based" permissions 
- hiding token requirements from **channel** context menu when channel is not token-gated

#### Platforms


- Android
- iOS


#### Areas that maybe impacted



### Steps to test


*Community*
- Open Status
- Open community with no token requirements but with token gated channels
- Open context-menu
- No "View token requirements" menu should be visible

*Channel*
- Open Status
- Open community
- Open non token gated chat
- Open context-menu
- No "View token requirements" menu should be visible


status: ready
